examples/python_bert_squad_example/BERT-Squad.ipynb

```diff
@@ -0,0 +1,284 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# BERT-SQuAD Inference Example with AMD MIGraphX"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This tutorial shows how to run the BERT-Squad model on ONNX-Runtime with MIGraphX backend."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Requirements "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip3 install -r requirements_migraphx.txt"
```

Comment:
can we rename the file name to "requrements_bertsquad.txt"? I think the requirements is for the bert model.

Reply:
Done